.azure-pipelines/client.test.live.yml

```diff
@@ -34,10 +34,16 @@ jobs:
       - task: Npm@1
         inputs:
           command: 'custom'
-          customCommand: 'run live-test-client -- -- -- --reporter mocha-junit-reporter'
+          customCommand: 'run live-test-client -- -- -- --reporter mocha-junit-reporter -f "Batch Receiver - Settle message Partitioned Queue: complete() removes message"'
         displayName: 'npm run live-test-client'
         env:
-          SERVICEBUS_CONNECTION_STRING: $(js-service-bus-test-connection-string)
+          AAD_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+          AAD_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+          AAD_SERVICEBUS_SECRET: $(aad-azure-sdk-test-client-secret)
```

Comment:
Are we changing `AAD_SERVICEBUS_SECRET` to `AAD_SECRET` ?
If yes, we need to change that in the code(testUtils.ts) as well.

Reply:
I will change both in a later PR

Reply:
I will probably name it `AAD_CLIENT_SECRET` by the way, since I've seen the name "client secret" used in docs.